layers/+vim/evil-better-jumper/README.org

```diff
@@ -0,0 +1,31 @@
+#+TITLE: evil-better-jumper layer
+#+TAGS: layer|vim
+
+* Table of Contents                                       :TOC_4_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+- [[#key-bindings][Key bindings]]
+
+* Description
+
+This layer adds support for [[https://github.com/gilbertw1/better-jumper][better-jumper]]. A configurable jump list
+implementation for Emacs that can be used to easily jump back to previous
+locations.
+
+** Features:
+- jump back and forth
+
+* Install
+
+To use this configuration layer, add it to your =~/.spacemacs=. You will need to
+add =evil-better-jumper= to the existing =dotspacemacs-configuration-layers=
+list in this file.
+
+* Key bindings
+
+| Key Binding | Description                 |
+|-------------+-----------------------------|
+| ~C-o~       | better-jumper-jump-backward |
+| ~C-i~       | better-jumper-jump-forward  |
```

Comment:
`C-i` sends `TAB` by default, it seems to be a terminal issue.
More info here: https://unix.stackexchange.com/questions/563469/conflict-ctrl-i-with-tab-in-normal-mode

In Spacemacs the variable: `dotspacemacs-distinguish-gui-tab` is `nil` by default.

When the variable is disabled, then `C-i` sends `TAB`.
In for example `.spacemacs`, `TAB` is bound to: `indent-for-tab-command`

When the variable is enabled and GUI Emacs is used,
then `C-i` sends:
```
 TAB                    ;; better-jumper-jump-forward
```

It should probably be mentioned in the readme.
Maybe it could say something like this.

```
* Key bindings

| Key Binding | Description                 |
|-------------+-----------------------------|
| ~C-o~       | better-jumper-jump-backward |

If the variable =dotspacemacs-distinguish-gui-tab= is non-nil.
And Spacemacs is running in a GUI (not a terminal).
Then ~C-i~ can be used to jump forward.
[[https://unix.stackexchange.com/questions/563469/conflict-ctrl-i-with-tab-in-normal-mode][Because terminals can't distinguish between ~C-i~ and ~TAB~.]]

| Key Binding | Description                 |
|-------------+-----------------------------|
| ~C-i~       | better-jumper-jump-forward  |

```

Could an option/addition be to bind `S-C-o` to `evil-jump-forward`.
It seems to be available. And it would match the behavior of 
for example undo which does the opposite when shift is held down:
In Emacs:
`C-/` undo
`S-C-/` redo

In other (most?) applications:
`C-z` undo
`S-C-z` redo

There are probably other examples, I just can't think of any at the moment.